### PR TITLE
Dev kheinicke add roles

### PR DESCRIPTION
**Pull request by MaxNoe**

### CODE REVIEW

File: database/__init__.py [hunk 1]
```diff
@@ -57,6 +62,28 @@ def pragma_on_connect(dbapi_con, con_record):
 )
 
 
+def access_required(name):
```
Comment (MaxNoe) [inline database/__init__.py]: This looks like it should go into `authentication.py`

File: database/__init__.py [hunk 2]
```diff
@@ -301,10 +328,15 @@ def login():
             flash('Invalid user or password', 'danger')
             return redirect(url_for('login', next=request.args.get('next')))
         login_user(user)
-        return redirect(request.args.get('next', url_for('index')))
+        return redirect(url_for(request.args.get('next', 'index')))
     return render_template('login.html', title='Login', form=form)
 
 
+@app.route('/config/pars/')
```
Comment (MaxNoe) [inline database/__init__.py]: What is this for?

File: database/__init__.py (hunk 1 above)
Comment (bixel) [reply database/__init__.py]: Agreed. Feels free to move it, I’m on vacation without my laptop right now 😝

File: database/__init__.py (hunk 2 above)
Comment (bixel) [reply database/__init__.py]: It is used by the PARS front end to configure UI elements. If we drop pars, we can drop this.

**Comment by bixel** [comment]
CORS is needed to develop PARS and the db simultaneously ok different ports of localhost. It is annoying... but it works.

**Comment by MaxNoe** [comment]
Can we merge just the roles part?

**Comment by bixel** [comment]
Yes, of course.

**Pull request merged**